kinbot/mess.py

```diff
@@ -1,17 +1,27 @@
-from __future__ import division
+from __future__ import print_function
+import logging
 import os
 import subprocess
 import time
 import pkg_resources
-import logging
+
 from kinbot import constants
 from kinbot import frequencies
-from kinbot import cheminfo
+from kinbot.uncertaintyAnalysis import UQ
+
 
 class MESS:
     """
-    Class that reads and write all MESS files and runs MESS
+    Class that read and writes MESS files for UQ analysis
+    Code is the same as mess.py exceppt that it loops to create 'n' files
+    and it randomizes the following parameters within the alloted UQ tolerance
+       1. Stationary point energy (E+ZPE, +/- 0.5 kcal/mol)
```

Comment:
The uncertainty ranges should be input parameters in parameter.py, these values are good as defaults.

Reply:
I believe those are the defaults in parameters.py, I can make that more clear in the comments.